docs-kits/kits/traceability-kit/openapi/block-notifications_2-0-0.yaml

```diff
@@ -0,0 +1,613 @@
+openapi: '3.1.1'

```

Comment:
shouldnt the yaml (since it is an openAPI) be located the the "root" openAPI folder?

Reply:
it seems that the file is two times in the repository, is there a need for this?